ddev/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Include ddev's source code when measuring its coverage ([#16057](https://github.com/DataDog/integrations-core/pull/16057))
 * Fix Github API search query ([#15943](https://github.com/DataDog/integrations-core/pull/15943))
 * Do not modify the Agent build name if provided by the user when running the e2e environments ([#16052](https://github.com/DataDog/integrations-core/pull/16052))
+* Also bump the Python version in the dependency provider when bumping the Python version ([#16070](https://github.com/DataDog/integrations-core/pull/16070))
```

Comment:
```suggestion
* Bump the Python version in the dependency provider when bumping the Python version ([#16070](https://github.com/DataDog/integrations-core/pull/16070))
```

Not sure ppl reading this will have enough context to understand the "also" part.

Reply:
sgtm!